contact-page.css

```diff
@@ -0,0 +1,118 @@
+@import url("https://fonts.googleapis.com/css?family=Archivo+Narrow|Cabin+Condensed|Roboto+Condensed|Varela+Round&display=swap");
+* {
+    font-family: "Varela Round", sans-serif;
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
+
+body {
+    font-size: medium;
+    text-align: center;
+}
+
+.nav-box {
+    position: relative;
+    margin: auto;
+    padding: 30px;
+    height: 22px;
+    width: 100%;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+}
+
+.navlist li {
+    display: inline;
+    list-style-type: none;
+    padding-right: 40px;
+}
+
+li a {
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: large;
+    color: black;
+}
+
+.banner-box {
+    position: relative;
+    text-align: center;
+    top: 5px;
+    padding: 200px;
+    margin-bottom: 30px;
+}
+
+.title-msg {
+    position: absolute;
+    padding: 0px;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+}
+
+h1 {
+    font-size: xx-large;
+    font-weight: bold;
+    text-transform: uppercase;
+}
+
+h2 {
+    font-size: x-large;
+    text-transform: capitalize;
+}
+
+.form-box {
+    top: 10px;
+    margin: 50px 200px;
+    justify-items: center;
+}
+
+.form-box #error-msg {
+    margin: 10px;
+    padding: 0;

```

Comment:
I've changed the padding to 10px to show that this is different from above. I set it as zero initially because this error message will only show when the form/field validation is invalid; usually, it's hidden. That's why the padding style set as 10px on the Javascript. I've also added a text error message here to give visual support.